Project.toml

```diff
@@ -1,29 +1,20 @@
-name = "REALMSupport"
-uuid = "926d8977-77e3-4bba-bc27-2e234c0c0038"
-authors = ["Jonathan Chang <jona125@gmail.com> and contributors"]
-version = "0.1.0"
```

Comment:
What's the reason for deleting these?

Reply:
I think this is automatic generated. Meaning some portion of the dependency package management is incorrect?

Reply:
I can't imagine what would do that (and haven't seen it happen before), but obviously not intentional. If you notice it again, let me know if you figure out what triggers it.